draft-js-resizeable-plugin/src/createDecorator.js

```diff
@@ -88,7 +88,7 @@ export default ({ config, store }) => (WrappedComponent) => class BlockResizeabl
     const doDrag = (dragEvent) => {
       let width = (startWidth + dragEvent.clientX) - startX;
       let height = (startHeight + dragEvent.clientY) - startY;
-      const block = store.getEditorRef().refs.editor;
+      const block = store.getEditorRef().editor;
```

Comment:
changing this throws an error for me on line 93 `clientWidth of undefined` - have you tested this?

Reply:
this happens both in firefox and chrome btw ...

Reply:
draft-js-plugin-editor is 2.0.4
draft-js-resizeable-plugin: 2.0.1

get error for `store.getEditorRef().refs.editor`
tested on chrome and firefox

Reply:
@freedomlang mind setting up a codesandbox for this? Would help a lot with fixing this

Reply:
the error is `Cannot read property 'editor' of undefined`